code/modules/mod/mod_control.dm

```diff
@@ -287,6 +281,14 @@
 		return
 	return ..()
 
+/obj/item/mod/control/attack_hand_secondary(mob/user, modifiers)
+	if(seconds_electrified && cell?.charge)
+		if(shock(user))
```

Comment:
can put this in the previous if

Reply:
this was copied over from attack_hand, do you want me to do it there too?

Reply:
sure